src/Components/Book.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+// import './Styles/book.css';
```

Comment:
* Kindly remove all commented code from your code base as it makes your code base bulky and unprofessional 🙏

Reply:
Sure 👍 
Thanks